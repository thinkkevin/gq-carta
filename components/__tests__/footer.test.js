import React from "react";
import Footer from "../footer";
import { shallow } from "enzyme";

describe("Test Footer", () => {
  it("validates the footer content using snapshot", () => {
    let comp = shallow(<Footer />);
    expect(comp.html()).toMatchSnapshot();
    expect(comp.exists(".footer")).toEqual(true); // check footer class is set
  });
});
