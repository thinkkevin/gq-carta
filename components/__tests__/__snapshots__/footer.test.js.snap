// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer validates the footer content using snapshot 1`] = `"<footer class=\\"footer\\"><div>Copyright Â© Kevin Guo</div><div>Powered by FourSquare API</div><div>Powered by Next.js framework</div></footer>"`;
